src/QirRuntime/test/QIR-static/qsharp/qir-gen.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.Quantum.Sdk/0.15.2102128445-alpha">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <QirGeneration>True</QirGeneration>
+  </PropertyGroup>
+
+</Project>

```

Comment:
File extension is `.csproj`. _C#_ rather than _Q#_ project?

Reply:
Yes, currently, Q# projects are built using dotnet.